### ISSUE TITLE
Arrow key navigation within menu bar

### ISSUE THREAD

**Issue by juliushaertl** [nextcloud/text#2827]
Navigation through formatting bar should be possible via arrow keys, tab should move into the formatting bar and next tab should move past it already

[w3.org/WAI/ARIA/apg/example-index/menu-button/menu-button-actions.html](https://www.w3.org/WAI/ARIA/apg/example-index/menu-button/menu-button-actions.html)

**Comment by juliushaertl** [comment]
Fixed by https://github.com/nextcloud/text/pull/4874
